config.schema.yaml

```diff
@@ -38,3 +38,32 @@
         type: "string"
         default: "testclientID"
         required: true
+
+  gcp_message_sensor:
+    description: "GCP sensor specific settings"
+    type: "object"
+    required: true
```

Comment:
The only minor thing, I think we can change both sensors configurations to `required: false`, since configuring 2 Kafka sensors is not a requirement for user.